core/spec/models/spree/order_spec.rb

```diff
@@ -203,15 +203,15 @@ def compute(computable)
     end
   end
 
-  describe '#ensure_line_item_variants_are_not_deleted' do
-    subject { order.ensure_line_item_variants_are_not_deleted }
+  describe '#ensure_line_item_variants_are_not_discontinued' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.